### DISCUSSION TITLE
Error with Component loader docx and pdf after upgrade Flowise

### DISCUSSION THREAD

**Discussion by claclin** [FlowiseAI/Flowise#1905] [Q&A]
Hi folks after upgrade Flowise to 1.6.0 now I can´t load pdf neither docx documents. For docx documents I get the following error:
"End of data reached (data length = 0, asked index=4) Corrupted zip?
![image](https://github.com/FlowiseAI/Flowise/assets/110524180/d9f1976c-7797-4030-8ce7-d6d273c3b65b)

And for the pdf document I get:
"Unknown error excepton: error PDFDocument: stream must have data.

![image](https://github.com/FlowiseAI/Flowise/assets/110524180/51f97dea-7029-4171-b052-2a6a9ef69d37)

Any help will be apprectiate..

**Comment by claclin** [comment]
Any help with this topic?

**Comment by prithvi151080** [comment]
any update on this issue...I am facing exactly the same issue.

Thnx in advance

**Reply by claclin** [reply]
Nothing here, it continues.

**Comment by prithvi151080** [comment]
@claclin: which browser you are using??? i faced this problem with edge...switch to chrome and I don't get this issue...plz try and let know...thnx

**Reply by claclin** [reply]
I am using Chrome

**Reply by prithvi151080** [reply]
it's an issue with some package conflict and not confined to flowise alone...many other users faced same issue with other apps...update all packages and retry in chrome...I was running flowise in edge...just switched to chrome and the problem disappeared... Hope this helps.

**Reply by claclin** [reply]
Thanks but I am not sure because I updated to the last version this weeend and it continues failing. I am using in docker so maybe something wrong with it...

**Reply by prithvi151080** [reply]
i am also running the entire setup on docker...

**Reply by claclin** [reply]
Then I don´t understand what is going wrong. I deployed the docker with an specific volume not the default, and if I try to upload files from folder it didn´t work either.

**Reply by prithvi151080** [reply]
I believe the specific volume for the docker that you are using is for persistent storage...are you using the same docker for vector or DB storage???? if yes plz make a dedicated docker for flowise only and then try in chrome...hope this helps

**Reply by claclin** [reply]
Yes I am using for persistent storage but I have created a dedicated docker for flowise only... I will try to deploy it again as default for checking it when I have time for it. Thanks.

**Reply by prithvi151080** [reply]
issue not sorted out...raised an issue with flowise..will update you if get any way around..

**Reply by prithvi151080** [reply]
solution found...use fresh docker image of version 1.6.4 to create flowise...

**Reply by claclin** [reply]
Thanks I will try it when I have a bit free time for it.

**Reply by claclin** [reply]
Hi, I have just make a fork of the new flowise version 1.7.2 and run the docker-compose up -d again with the new fork. Trying to upload the pdf or docx file and now it works. Thanks

**Comment by claclin** [comment]
Fixed after update to the latest version

**Comment by JNNelson** [comment]
Same problem here.
version 1.8.3 or 1.7.2.
Docker 26.1.4
However, the error only occurs in Docker. If you install directly on the VPS there is no error.
@claclin
